### PR TITLE
2.3-bet-3334-upgrade-details

### PR DESCRIPTION
**Pull request by resmaa**
https://spectrocloud.atlassian.net/browse/BET-3334
 
"Event details" added as part of: 
Cluster -Troubleshooting
https://docs.spectrocloud.com/clusters/#troubleshooting

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/599#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/04-clusters.md [hunk 1]
```diff
@@ -123,6 +123,25 @@ Spectro Cloud maintains specific milestones in a lifecycle and presents them as
...
+* Changes in availability zones
+* Changes in instance types
+* Certificate renewal and many more..
+
+Palette also keeps track of node's machine health and will relaunch the node when the machine health check fails. 
+Palette keeps track of the reason that triggered the rolling upgrade on the nodes in the cluster and is made accessible under **Cluster Overview > Upgrade details**.
+
+Note:
+* For relaunching a node healthcheck timeout is 10 mins. Hence, if node’s kubelet stopped working and doesnot come up in 10 mins a new node will be launched.  
+* For network unavailability the time delay is 10 mins. Hence, if new node is launching and it doesn’t get ready in 30 mins the failed node will be killed and replaced with a new node.
```
Comment (sadysnaat) [inline packages/docs/content/04-clusters.md:143]: we should separate 

```
For network unavailability the time delay is 10 mins. Hence, if new node is launching and it doesn’t get ready in 30 mins the failed node will be killed and replaced with a new node.
```

two parts
```
For network unavailability the time delay is 10 mins. 

For new node launching and it doesn’t get ready in 30 mins the failed node will be killed and replaced with a new node.
```

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/599#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/599#pullrequestreview-894287626" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**